src/clusterfuzz/_internal/fuzzing/fuzzer_selection.py

```diff
@@ -118,11 +118,12 @@ def get_fuzz_task_payload(platform=None):
   if base_platform != platform:
     platforms.append(base_platform)
 
-  query = data_types.FuzzerJobs.query()
   if environment.is_local_development():
-    query = query.filter(data_types.FuzzerJobs.platform.IN(platforms))
+    query = data_types.FuzzerJob.query()
+    query = query.filter(data_types.FuzzerJob.platform.IN(platforms))
```

Comment:
I like the refactor idea but would prefer to avoid the negative boolean.

what do you think about the introduction of `is_production()` helper function that returns the same as the condition in your original suggestion?